tests/Module/Fixture/DisplayUserEvents.php

```diff
@@ -85,6 +85,67 @@ public function saveUserEvents(int $owner, array $events)
 
         return ['events' => $events, 'eventsId' => $eventsId];
     }
+
+    public function haveIncrementalEvents(int $owner)
+    {
+        $events = ['incremental_event_first', 'incremental_events_second'];
+        $date = (new \DateTime())->format('Y-m-d');
+
+        foreach(range(0,1) as $i) {
+            $this->laravel->haveRecord(
+                self::TABLE_DISPLAYY_USER_EVENTS,
+                [
+                    'user_id' => $owner,
+                    'event_name' => $events[$i],
+                    'type' => 'incremental',
+                    'order' => 0,
+                    'created_at' => $date,
+                    'updated_at' => $date
+                ]);
+            $this->laravel->seeRecord(
+                self::TABLE_DISPLAYY_USER_EVENTS,
+                [
+                    'user_id' => $owner,
+                    'event_name' => $events[$i],
+                    'type' => 'incremental',
+                    'order' => 0,
+                    'created_at' => $date,
+                    'updated_at' => $date
+                ]);
+        }
+
+        return $events;
+    }
+
+    public function haveSummarizableEvents(int $owner)
+    {
+        $date = (new \DateTime())->format('Y-m-d');
+
+        foreach(range(0,1) as $i) {
+            $this->laravel->haveRecord(
+                self::TABLE_DISPLAYY_USER_EVENTS,
+                [
+                    'user_id' => $owner,
+                    'event_name' => 'summarizable_event',
+                    'type' => 'summarizable',
+                    'order' => 0,
+                    'created_at' => $date,
+                    'updated_at' => $date
+                ]);
+            $this->laravel->seeRecord(
+                self::TABLE_DISPLAYY_USER_EVENTS,
+                [
+                    'user_id' => $owner,
+                    'event_name' => 'summarizable_event',
+                    'type' => 'summarizable',
+                    'order' => 0,
+                    'created_at' => $date,
+                    'updated_at' => $date
+                ]);
+        }
+
+        return 'summarizable_event';
```

Comment:
In the first function you're returning the array, here you're returning the string. Please, return here the list of ids of created records.